README.md

```diff
@@ -4,17 +4,17 @@ A reference library for building Javascript applications on top of the Enceeper
 
 ## Introduction
 
-The [Enceeper app](https://github.com/enceeper/enceeper) and the [Enceeper service](https://www.enceeper.com/) can be used to securely store and retrieve credentials (usernames, passwords, API keys etc). The Enceeper app can be divided into two parts:
+The [Enceeper app](https://github.com/enceeper/enceeper) and the [Enceeper service](https://www.enceeper.com/) are used to securely store and retrieve credentials (usernames, passwords, API keys etc). The Enceeper app is divided into two parts:
 - The User Interface (UI) that handles all user input
 - The core functionality and the utilization of the Enceeper service
 
-For the later we have created this repository, to make it a seperate project for easier integration (via npm) to other solutions.
+For the latter, use this repository which serves as a basis project for easier integration (via npm) to other solutions.
```

Comment:
Should we add at the end: "Our own cross-platform Enceeper app is based on this library."

Reply:
Sure thing. I suppose u will add it 👍

Reply:
On a second view..it looks like te same line exists on the opening sentence. "A reference library for building Javascript applications on top of the Enceeper service. Our own cross-platform Enceeper app is using this library."

I'm thinking to remove "own" from there and don't add the line here. Thoughts?

Reply:
Own should be removed. If I remove the same sentence do you think it is clear that this library is used by the Enceeper app?

Reply:
To be clear. 
1. If you keep the opening sentence, i don't think you should add the same line on this point.
2. If you remove it from the opening sentence, then it makes sense to add it here.

generally speaking i believe anything inside this project (enceeper) kind of implies that is also being utilised.